tests/controllers/UsersControllerTest.php

```diff
@@ -0,0 +1,166 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: liam
+ * Date: 10/09/16
+ * Time: 12:54
+ */
+
+namespace JoindinTest\Controller;
+
+
+class UsersControllerTest extends \PHPUnit_Framework_TestCase
+{
+
+    /**
+     * Ensures that if the deleteUser method is called and no user_id is set,
+     * an exception is thrown
+     *
+     * @return void
+     *
+     * @test
+     * @expectedException \Exception
+     * @expectedExceptionMessage You must be logged in to delete data
+     */
+    public function testDeleteUserWithNoUserIdThrowsException(){
```

Comment:
There was a very good reason for this - I was being lazy :-).

Have sorted, will look at those return codes, and as long as they don't break anything (I don't *think* they will, but will test), then I'll update them.